### ISSUE TITLE
Pasting from Word - paragraph inside heading

### ISSUE THREAD

**Issue by mwtorkowski** [froala/wysiwyg-editor#304]
After pasting content from certain Word documents, headings include content wrapped in paragraph - this structure prevents "block style" button (dropdown) to actually work - there's no way to turn such heading into paragraph (clean formatting button won't work here either) - user would have to resort to html view to fix it.

Honestly, I don't know how common this use case is, but seems like something that should be handled, either by improving pasting from Word functionality OR by making actual "clear formatting" button that would effectively turn selected content to plain text (like clear formatting button on gmail).

Bottom line is there should be an easy (non-html way) to recover from any weird formatting situations.

**Comment by stefanneculai** [comment]
@mwtorkowski Could you send a document explaining what exactly you copy? Also, what browser are you trying it on? We'd like to get this fixed but we can't replicate it.

**Comment by dianaprajescu** [comment]
Closing this issue due to lack of information.
